KuleumBridge/TabbarView.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+struct TabbarView: View {
+    @State private var selectedIdx = 0
+
+    let tabbarText = ["홈", "맛집", "공지사항"]
+
+    var body: some View {
+        VStack {
+            ZStack {
+                switch selectedIdx {
+                case 0:
+                    // TODO: 홈 뷰 연결
+                    ContentView()
+                case 1:
+                    TastePlaceView()
+                default:
+                    // TODO: 공지사항 뷰 연결
+                    ContentView()
+                }
+            }
```

Comment:
```suggestion
                switch selectedIdx {
                case 0:
                    // TODO: 홈 뷰 연결
                    ContentView()
                case 1:
                    TastePlaceView()
                case 2:
                    // TODO: 공지사항 뷰 연결
                    ContentView()
                default:
                    SomethingView()
                }
            }
```
이럴 때는 default로 다른 걸 두고 case에는 모든 index를 넣어두는게 좋습니다.